frontend/index.jsx

```diff
@@ -33,8 +33,11 @@ class App extends React.Component {
       imageFeed: [],
       subscribers: [],
       aesKey: null,
-      tab: 'my'
+      tab: 'my',
+
+      isLoading: false
     }
+    this.readSubscribersImages = this.readSubscribersImages.bind(this);
```

Comment:
careful. you are reintroducing things, which were already fixed

Reply:
no I just pulled